src/d/a/d_a_alink.cpp

```diff
@@ -19192,24 +19129,14 @@ asm void mDoExt_MtxCalcAnmBlendTbl::getAnm(int param_0) {
 #pragma pop
 
 /* 800D00D0-800D00DC 0CAA10 000C+00 1/1 0/0 0/0 .text            ChkRoofHit__9dBgS_AcchCFv */
-#pragma push
-#pragma optimization_level 0
-#pragma optimizewithasm off
-asm void dBgS_Acch::ChkRoofHit() const {
-    nofralloc
-#include "asm/d/a/d_a_alink/ChkRoofHit__9dBgS_AcchCFv.s"
+bool dBgS_Acch::ChkRoofHit() const {
+    return m_flags & ROOF_HIT ? true : false;
```

Comment:
```suggestion
    return m_flags & ROOF_HIT;
```
There are a few other instances of this in this PR